### PR TITLE
OAUTH2-113 Test update

### PR DESCRIPTION
**Pull request by csierra**

### CODE REVIEW

File: modules/apps/oauth2-provider/oauth2-provider-jsonws/src/main/java/com/liferay/oauth2/provider/jsonws/internal/service/access/policy/OAuth2JSONWSSAPEntryActivator.java [hunk 1]
```diff
@@ -81,7 +81,7 @@ protected void deactivate() {
 
 	private static final String[][] _SAP_ENTRY_OBJECT_ARRAYS = {
 		{
-			"OAUTH2_documents_download",
+			"OAUTH2_everything.readonly.download.documents",
```
Comment (stian-sigvartsen) [inline modules/apps/oauth2-provider/oauth2-provider-jsonws/src/main/java/com/liferay/oauth2/provider/jsonws/internal/service/access/policy/OAuth2JSONWSSAPEntryActivator.java]: Do you think there is scope for a different scope alias (i.e. different assets/ services) in future starting with "everything.readonly.download."?

Comment (csierra) [reply modules/apps/oauth2-provider/oauth2-provider-jsonws/src/main/java/com/liferay/oauth2/provider/jsonws/internal/service/access/policy/OAuth2JSONWSSAPEntryActivator.java]: I don't know... I just transposed it from `documents_download` to `download.documents`

Comment (csierra) [reply modules/apps/oauth2-provider/oauth2-provider-jsonws/src/main/java/com/liferay/oauth2/provider/jsonws/internal/service/access/policy/OAuth2JSONWSSAPEntryActivator.java]: actually, as we discussed, I would expect this to go away in future versions when the API gets better

Comment (stian-sigvartsen) [reply modules/apps/oauth2-provider/oauth2-provider-jsonws/src/main/java/com/liferay/oauth2/provider/jsonws/internal/service/access/policy/OAuth2JSONWSSAPEntryActivator.java]: Yes when there are more SAP profiles covering suites. Ok, will keep you "*.download.documents" scheme in my upcoming pull. Thanks!

**Comment by stian-sigvartsen** [comment]
Integrated into my branch, thanks! See csierra/liferay-portal/pull/422

**Pull request closed without merging**